common/components/SendButtonFactory/OfflineBroadcast.tsx

```diff
@@ -20,7 +20,7 @@ export const OfflineBroadcast = connect((state: AppState) => ({ offline: getOffl
 const BroadCast: React.SFC<{}> = () => (
   <p>
     To broadcast this transaction, paste the above into the{' '}
-    <Link to="pushTx">Broadcast Transaction tab</Link> or{' '}
+    <Link to="../pushTx">Broadcast Transaction tab</Link> or{' '}
```

Comment:
Don't do `..`, we just want it to be `/pushTx` so that it goes to the root. `..` wouldn't work if it were nested 2 levels deep, e.g. `/account/send/confirm` or something like that.

Reply:
It didn't occur to me that it would need to be referenced from anywhere else. Thanks 💃